lib/CultureFeed/Cdb/Data/Language.php

```diff
@@ -9,7 +9,7 @@ final class CultureFeed_Cdb_Data_Language implements CultureFeed_Cdb_IElement
     private ?string $language;
     private ?string $type;
 
-    public function __construct($language = null, $type = null)
```

Comment:
This is probably an error that was already present in the codebase, so not worth fixing, but the getters only return string instead of nullable strings, while you can construct it with nulls.

Reply:
I changed the return type to also include nullable. Static analysis can give better feedback when nullable is also specified.